dbms/src/Functions/FunctionsJSON.h

```diff
@@ -279,6 +279,7 @@ class FunctionJSON : public IFunction
 
 
 struct NameJSONHas { static constexpr auto name{"JSONHas"}; };
+struct NameIsValidJSON { static constexpr auto name{"isValidJSON"}; };
```

Comment:
I think the name `JSONIsValid()` would be more consistent with other names.

Reply:
Ok. BTW, isValidJSON is consistent with isValidUTF8.